diff -Naur eglexternalplatform-1.2.old/meson.build eglexternalplatform-1.2/meson.build
--- eglexternalplatform-1.2.old/meson.build	2024-09-04 09:42:32.677731661 +0200
+++ eglexternalplatform-1.2/meson.build	2024-09-04 09:42:53.143945396 +0200
@@ -4,7 +4,8 @@
 pkg = import('pkgconfig')
 pkg.generate(filebase: 'eglexternalplatform',
   name: 'EGL External Platform interface',
-  description: 'EGL External Platform interface')
+  description: 'EGL External Platform interface',
+  install_dir: get_option('datadir') / 'pkgconfig')
 
 install_headers(
   'interface/eglexternalplatform.h',
